Dockerfile

```diff
@@ -7,25 +12,87 @@ RUN apt-get update && apt-get install -y wget \
     git \
     python
 
-ADD . /opt/apalache/
+ADD ./3rdparty /opt/apalache/3rdparty
 WORKDIR /opt/apalache/
 
 # Workaround for Surefire not finding ForkedBooter
 # (see https://stackoverflow.com/questions/53010200/maven-surefire-could-not-find-forkedbooter-class)
 ENV _JAVA_OPTIONS="-Djdk.net.URLClassPath.disableClassPathURLCheck=true"
```

Comment:
@konnov or @andrey-kuprianov do you know if this is needed in theory for building our 3rd party deps?

Reply:
The installation script `3rdparty/install-local.sh` publishes `z3.jar` in the local maven repo. Apalache needs that jar to compile the code.

Reply:
Sorry, I think I'm missing the connection. Is the surefire workaround here related to the `z3.jar`?

Reply:
Oh. The Surefire option is probably needed for building the project inside docker. It has been for several years there, so probably it is not needed anymore.

Reply:
Turns out we don't need it. I ran the integration tests against the app image built with this flag removed and everything worked as expected.